webrender/src/renderer.rs

```diff
@@ -666,7 +665,7 @@ struct SourceTextureResolver {
 }
 
 impl SourceTextureResolver {
-    fn new(device: &mut Device<back::Backend, hal::Graphics>) -> SourceTextureResolver {
+    fn new<B: hal::Backend>(device: &mut Device<B, hal::Graphics>) -> SourceTextureResolver {
```

Comment:
I believe, technically, all these types (like `SourceTextureResolver`) need a `PhantomData<B>` member. You can even have `B` unconstrained on the struct itself, but the implementations will become less noisy